### PR TITLE
Add a data attribute to the created style tag.

### PR DESCRIPTION
**Pull request by milesj**
I'm working on a React style abstraction layer and it's quite hard to test against Glamor as style tags are not unique. This adds a data attribute so that I may find them easily.

https://github.com/milesj/aesthetic

Edit: Looks like my editor strips tailing spaces (you should be doing the same). I can redo this PR if it's too much trouble.

### CODE REVIEW

File: src/sheet.js [hunk 1]
```diff
@@ -46,25 +46,26 @@ const isBrowser = typeof window !== 'undefined'
...
     div.innerHTML = '<!--[if lt IE 10]><i></i><![endif]-->'
     return div.getElementsByTagName('i').length === 1
-  }  
+  }
 })()
 
 function makeStyleTag() {
   let tag = document.createElement('style')
   tag.type = 'text/css'
+  tag.setAttribute('data-glamor', '')
```
Comment (milesj) [inline src/sheet.js]: Change here.

File: tests/index.js [hunk 2]
```diff
@@ -659,26 +659,28 @@ describe('StyleSheet', () => {
...
   afterEach(() => {
     sheet.flush()
   })
 
-  it('can initialize', () => {    
-    expect([ ...document.styleSheets ].filter(s => s.ownerNode === sheet.tags[0]).length).toEqual(1)    
+  it('can initialize', () => {
+    expect([ ...document.styleSheets ].filter(s => s.ownerNode === sheet.tags[0]).length).toEqual(1)
+
+    expect(sheet.tags[0].hasAttribute('data-glamor')).toEqual(true)
```
Comment (milesj) [inline tests/index.js]: And here.

**Comment by threepointone** [comment]
The whitespace seems to have broken one of my tests, so I can't use this PR. I just manually copied your changes in though https://github.com/threepointone/glamor/commit/518e63ca1a12476f40dcb665d32b3b4083e6dde2 this should go out in the next release 

be warned - glamor uses multiple tags as the rule space grows, this may or may not be relevant to your library. best of luck!

**Comment by threepointone** [comment]
this is live in 2.20.17

**Comment by milesj** [comment]
Awesome, thanks for pulling it in!

**Pull request closed without merging**